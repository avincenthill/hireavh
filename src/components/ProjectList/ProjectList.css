.projects-container {
  display: flex;
  align-items: center;
  align-content: center;
  justify-content: center;
  flex-wrap: wrap;
  margin-top: var(--s-s);
  margin-bottom: var(--s-s);
}
