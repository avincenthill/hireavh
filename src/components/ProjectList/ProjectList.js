import "./ProjectList.css";
import Project from "components/Project/Project";
import React from "react";

class ProjectList extends React.Component {
  renderProjects = (projects) => {
    if (projects) {
      return projects.map((project, index) => {
        return <Project project={project} key={index}></Project>;
      });
    }
  };

  render() {
    return (
      <div>
        <h2 className="projectlist-title">{this.props.title}</h2>
        <div className="projects-container">
          {this.props ? this.renderProjects(this.props.projects) : null}
        </div>
      </div>
    );
  }
}

export default ProjectList;
