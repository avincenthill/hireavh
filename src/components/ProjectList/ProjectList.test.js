import ProjectList from "./ProjectList";
import React from "react";
import renderer from "react-test-renderer";

describe("ProjectList", () => {
  describe("snapshot", () => {
    it("should match the last saved snapshot", () => {
      const tree = renderer.create(<ProjectList />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
