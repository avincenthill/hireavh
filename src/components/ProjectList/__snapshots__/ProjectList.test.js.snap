// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectList snapshot matches the last saved snapshot 1`] = `
<div>
  <h2
    className="projectlist-title"
  />
  <div
    className="projects-container"
  />
</div>
`;

exports[`ProjectList snapshot should match the last saved snapshot 1`] = `
<div>
  <h2
    className="projectlist-title"
  />
  <div
    className="projects-container"
  />
</div>
`;
